README.md

```diff
@@ -38,7 +38,27 @@ lineage:
     remote-url: https://github.com/felddy/extra-skel-sauce.git
 ```
 
-## Sample GitHub Actions workflow ##
+## Usage ##
+
+### Inputs ###
+
+| Name | Description | Interpreted Type | Default | Required |
+|------|-------------|------------------|---------|:--------:|
+| access_token | GitHub personal access token (see [GitHub's documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)). | `string` | n/a | yes |
+| mask_non_public_repos | Control the visibility of private repo names in the logs. | [`boolean`](https://yaml.org/spec/1.2.2/#1032-tag-resolution) | `true` | no |
+| public_repos_only | Control non-public repository processing. | [`boolean`](https://yaml.org/spec/1.2.2/#1032-tag-resolution) | `true` | no |
+| repo_query | GitHub search query to find repositories for pull requests (e.g. "org:cisagov archived:false"). | `string` | n/a | yes |
```

Comment:
* I think "private" is a much more succinct way of saying "non-public".
* Your descriptions for the boolean values tell me that they "control" something but don't tell me when i should use `true` and when i should use `false`.
* The description for `repo_query` was a little awkward and unclear, so I touched it up.
```suggestion
| mask_private_repos | Whether to mask the names of private repositories in the logs. | [`boolean`](https://yaml.org/spec/1.2.2/#1032-tag-resolution) | `true` | no |
| include_private_repos | Whether to process private repositories. | [`boolean`](https://yaml.org/spec/1.2.2/#1032-tag-resolution) | `false` | no |
| repo_query | GitHub search query to use when finding repositories for which to create pull requests (e.g. "org:cisagov archived:false"). | `string` | n/a | yes |
```

Reply:
OK, I wasn't thinking of the internal repos.  What about the two other bullet points?

Reply:
Ah right, those descriptions were a bit rough so I'm happy to improve them.

Reply:
Would you take a look at 4afa7c4 and let me know what you think?